plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -67,9 +71,9 @@ protected void execute(IProgressMonitor monitor)
     // The project ID is currently necessary due to tool bugs.
     properties.put("application-id", artifactId);
     properties.put("useObjectify", "false");
-    properties.put("useEndpoints1", "false");
-    properties.put("useEndpoints2", "false");
-    properties.put("useAppEngineApi", "false");
+    properties.put("useEndpoints1", appEngineLibraryIds.contains("appengine-endpoints"));
```

Comment:
This value needs to be a string, not a boolean. Don't ask me to explain why but I spent a day debugging that. :-( Ditto below.